src/videojs-contrib-hls.js

```diff
@@ -161,6 +167,10 @@ Hls.STANDARD_PLAYLIST_SELECTOR = function() {
       // since the playlists are sorted in ascending order by
       // bandwidth, the first viable variant is the best
       if (!bandwidthBestVariant) {
+        if (i !== 0 &&
+            sortedPlaylists[i - 1].attributes.BANDWIDTH === variant.attributes.BANDWIDTH) {
+          continue;
+        }
```

Comment:
This might be easier done as a `filter` to get all of the renditions with the same (highest) bandwidth and then taking the very first element.